src/components/searchBar/citiesSearch/CitiesSearch.jsx

```diff
@@ -1,11 +1,7 @@
-import React from 'react'
+import React from 'react';
 
 function CitiesSearch() {
-    return (
-        <div>
-            
-        </div>
-    )
+  return <div></div>;
 }
 
-export default CitiesSearch
+export default CitiesSearch;
```

Comment:
This _SearchedCities_ name will be better. Your component name is Incomprehensible.